src/main/java/Todo.java

```diff
@@ -0,0 +1,10 @@
+public class Todo extends Task{
+    public Todo(String description) {
+        super(description);
+    }
+
+    @Override
+    public String toString() {
+        return "[T]" + super.toString();
+    }
+}
```

Comment:
Good job, readable and short and succinctly written class, with good use of @Override keyword to tell the reader that this method is overridden